esignet-default.properties

```diff
@@ -28,31 +28,59 @@
 mosip.esignet.misp.license.key=${mosip.esignet.misp.key}
-mosip.esignet.amr-acr-mapping-file-url=${config.server.file.storage.uri}amr-acr-mapping.json
+mosip.esignet.amr-acr-mapping-file-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/amr-acr-mapping.json
 mosip.esignet.auth-txn-id-length=10
 mosip.esignet.supported-id-regex=\\S*
 # Generated ID and access tokens 'exp' depends on the below properties, default value is 1-hour
-mosip.esignet.id-token-expire-seconds=86400
-mosip.esignet.access-token-expire-seconds=86400
+mosip.esignet.id-token-expire-seconds=3600
+mosip.esignet.access-token-expire-seconds=3600
 # By default, only 2 link codes can be active, and the time period it can be active is defined here, default value is 1 minute
 mosip.esignet.link-code-expire-in-secs=60
 # Number of link code allowed to be generated in a transaction, the default value is 10
 mosip.esignet.generate-link-code.limit-per-transaction=10
 # Time to complete consent after successful authentication, the default value is 120
-mosip.esignet.authentication-expire-in-secs=86400
+mosip.esignet.authentication-expire-in-secs=120
 
+# Auth challenge type & format mapping. Auth challenge length validations for each auth factor type.
+mosip.esignet.auth-challenge.OTP.format=alpha-numeric
+mosip.esignet.auth-challenge.OTP.min-length=6
+mosip.esignet.auth-challenge.OTP.max-length=6
+
+mosip.esignet.auth-challenge.PWD.format=alpha-numeric
+mosip.esignet.auth-challenge.PWD.min-length=8
+mosip.esignet.auth-challenge.PWD.max-length=30
+
+mosip.esignet.auth-challenge.BIO.format=encoded-json
+mosip.esignet.auth-challenge.BIO.min-length=5000
+mosip.esignet.auth-challenge.BIO.max-length=300000
+
+mosip.esignet.auth-challenge.WLA.format=jwt
+mosip.esignet.auth-challenge.WLA.min-length=100
+mosip.esignet.auth-challenge.WLA.max-length=1500
+
+mosip.esignet.auth-challenge.KBA.format=base64url-encoded-json
+mosip.esignet.auth-challenge.KBA.min-length=50
+mosip.esignet.auth-challenge.KBA.max-length=500
+
+mosip.esignet.auth-challenge.PIN.format=number
+mosip.esignet.auth-challenge.PIN.min-length=4
+mosip.esignet.auth-challenge.PIN.max-length=4
+
+
+# Endpoints required to have oauth-details-hash and oauth-details-key HTTP header
 mosip.esignet.header-filter.paths-to-validate={'${server.servlet.path}/authorization/send-otp', \
   '${server.servlet.path}/authorization/authenticate', \
   '${server.servlet.path}/authorization/v2/authenticate', \
   '${server.servlet.path}/authorization/v3/authenticate', \
   '${server.servlet.path}/authorization/auth-code'}
 
-mosip.esignet.captcha.required=send-otp,pwd,bio,pin
-
-mosip.esignet.captcha.required.auth-factors={'PWD'}
-mosip.esignet.captcha.required.auth-factors={}
+#This property is used for captcha validation and allowed values are send-otp, pwd and kba.
+#captcha validation is enabled for send-otp, pwd and kba.
+mosip.esignet.captcha.required=send-otp,pwd
```

Comment:
We should disable captcha, as its performance testing

Reply:
disabled